Neos.ContentRepository.LegacyNodeMigration/Classes/Command/CrCommandController.php

```diff
@@ -38,16 +38,16 @@
 class CrCommandController extends CommandController
 {
     public function __construct(
-        private readonly Connection $connection,
-        private readonly Environment $environment,
+        private readonly Connection                  $connection,
+        private readonly Environment                 $environment,
         private readonly PersistenceManagerInterface $persistenceManager,
-        private readonly AssetRepository $assetRepository,
-        private readonly ResourceRepository $resourceRepository,
-        private readonly ResourceManager $resourceManager,
-        private readonly PropertyMapper $propertyMapper,
-        private readonly ContentRepositoryRegistry $contentRepositoryRegistry,
-        private readonly SiteRepository $siteRepository,
-        private readonly ProjectionReplayServiceFactory $projectionReplayServiceFactory,
+        private readonly AssetRepository             $assetRepository,
+        private readonly ResourceRepository          $resourceRepository,
+        private readonly ResourceManager             $resourceManager,
+        private readonly PropertyMapper              $propertyMapper,
+        private readonly ContentRepositoryRegistry   $contentRepositoryRegistry,
+        private readonly SiteRepository              $siteRepository,
+        private readonly ProjectionServiceFactory    $projectionServiceFactory,
```

Comment:
Not a big issue ofc, but I don't like the style of aligning parameters (mostly because it leads to multi-line diffs like this one) and we don't do that in other places

Reply:
I am not sure this is an issue at all. The export does not contain neither workspaceName nor contentStreamId any more so those values are only set on the exported events as temporary values so we can use the default eventObjects.